contracts/staking/contracts/src/vaults/EthVault.sol

```diff
@@ -0,0 +1,110 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+
+import "@0x/contracts-utils/contracts/src/LibSafeMath.sol";
+import "../interfaces/IEthVault.sol";
+import "./MixinVaultCore.sol";
+
+
+/// @dev This vault manages ETH.
+contract EthVault is
+    Authorizable,
+    IEthVault,
+    IVaultCore,
+    MixinVaultCore
+{
+
+    using LibSafeMath for uint256;
+
+    // mapping from Owner to ETH balance
+    mapping (address => uint256) internal balances;
```

Comment:
Nit: `_balances`

Reply:
Note: we chatted offline. This will be done in a separate PR.